apps/web/pages/event-types/index.tsx

```diff
@@ -929,7 +929,7 @@
       !!orgBranding && !document.cookie.includes("calcom-profile-banner=1") && !user?.completedOnboarding
     );
```

Comment:
<i class='graphite__hidden'>[Re: lines 928 to 930]<br/></i>

NIT - this entire set should be in a separate useEffect as it has different dependancies  compared to the show intercom bubble.

<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/calcom/cal.com/10546?utm_source=unchanged-line-comment">Graphite</a>.</span>

Reply:
Good catch, done